### PR TITLE
release notes for storage release 2023-03-03

### PR DESCRIPTION
**Pull request by problame**
@vadim2404 , @sharnoff  , feel free to amend this PR as needed.
Previews:
https://neon-next-git-problame-release-notes-2023-03-03-neondatabase.vercel.app/docs/release-notes/2023-03-03
https://neon-next-git-problame-release-notes-2023-03-03-neondatabase.vercel.app/docs/reference/pg-extensions

### CODE REVIEW

**Review by sharnoff — APPROVED** [review]
lgtm - my PRs didn't affect code that was actually in use in prod, so should be ok to omit.

**Comment by vadim2404** [comment]
There were two new extensions: pgTAP and rum.

**Review by danieltprice — DISMISSED** [review]

**Review by danieltprice — DISMISSED** [review]

**Review by danieltprice — APPROVED** [review]

File: content/release-notes/2023-03-03.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ label: 'Storage'
 
 - Pageserver: A system metric that monitors physical data size overflowed when a garbage collection operation was performed on an evicted data layer.
 - Pageserver: An index upload was skipped when a compaction operation did not perform an on-demand download from storage. With no  on-demand downloads, the compaction function would exit before scheduling the index upload.
+- Added support for the PostgreSQL `run` and `pgTAP` extensions. For more information about PostgreSQL extensions supported by Neon, see [PostgreSQL extensions](https://neon.tech/docs/reference/pg-extensions/).
```
Comment (vadim2404) [inline content/release-notes/2023-03-03.md:9]: rum should be

Comment (danieltprice) [reply content/release-notes/2023-03-03.md]: fixed. ty!

**Review by danieltprice — DISMISSED** [review]

**Review by danieltprice — APPROVED** [review]

**Pull request merged**